### PR TITLE
feat(proposals): add energy use elements proposal

### PR DESCRIPTION
**Pull request by macintoshpie**
Also see these [slides](https://docs.google.com/presentation/d/1-JSfXUhc68JUhGdDLkGgSCkLt_xSW1uMVEV872pEgm0/edit?usp=sharing) on how we plan on having BuildingSync model Energy Use and EUI

### CODE REVIEW

File: proposals/2020/Add Energy Use Elements.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- `auc:BuildingEnergyUse`
+- `auc:BuildingEnergyUseIntensity`
+- `auc:ImportedEnergyConsistentUnits`
+- `auc:OnsiteEnergyProductionConsistentUnits`
+- `auc:ExportedEnergyConsistentUnits`
+- `auc:NetIncreaseInStoredEnergyConsistentUnits`
+
+## Justification
+
+Standard 211 sections 5.2.3 and 5.2.3.1 require Building energy use and use intensity (ie it's non-optional in this proposal). Adding the other energy elements could provide an easier way to validate calculations, a more direct mapping to the 211 spreadsheet, and a more understandable breakdown of how 211 models EUI for users.
```
Comment (corymosiman12) [inline proposals/2020/Add Energy Use Elements.md:15]: Change to:

Standard 211 refers to ASHRAE 105 for defining specific terms related to energy use.  Within ASHRAE 105, there is a clear distinction between Building Energy and Site Energy, as depicted by ASHRAE 105 Figure 5.6.  Additionally defined in ASHRAE 105 are the following intermediate calculation concepts:
- Imported Energy
- Exported Energy
- On-Site Renewable Energy Production
- Net Increase in Stored Imported Energy

Currently, the `auc:AllResourceTotal` complex type includes:
- `auc:SiteEnergyUse`
- `auc:SiteEnergyUseIntensity`

We want to shore up this representation by distinctly aligning BuildingSync elements with ASHRAE 105, with the intention of making the distinction between Building and Site energy more clear, as well as having a referenceable standard by which users can refer for making energy calculations for their site or building.

File: BuildingSync.xsd [hunk 2]
```diff
@@ -4030,7 +4030,7 @@
       <xs:element ref="auc:ResourceBoundary" minOccurs="0"/>
       <xs:element name="SiteEnergyUse" minOccurs="0">
         <xs:annotation>
-          <xs:documentation>The annual amount of all the energy the premises consumes onsite, as reported on the utility bills. (kBtu)</xs:documentation>
+          <xs:documentation>The annual amount of all the energy the premises consumes onsite, as reported on the utility bills. Calculated as imported energy - exported energy - net increase in stored energy (per ASHRAE ASHRAE 105 Figure 5.6). (kBtu)</xs:documentation>
```
Comment (markborkum) [inline BuildingSync.xsd:4033]: Typo "ASHRAE ASHRAE"

File: BuildingSync.xsd [hunk 3]
```diff
@@ -4030,7 +4030,7 @@
       <xs:element ref="auc:ResourceBoundary" minOccurs="0"/>
       <xs:element name="SiteEnergyUse" minOccurs="0">
         <xs:annotation>
-          <xs:documentation>The annual amount of all the energy the premises consumes onsite, as reported on the utility bills. (kBtu)</xs:documentation>
+          <xs:documentation>The annual amount of all the energy the premises consumes onsite, as reported on the utility bills. Calculated as imported energy - exported energy - net increase in stored energy (per ASHRAE 105-2014 Figure 5.6). (kBtu)</xs:documentation>
```
Comment (macintoshpie) [inline BuildingSync.xsd:4033]: @corymosiman12 can you confirm this is the right "name" of the standard?

Comment (corymosiman12) [reply BuildingSync.xsd:4033]: Yes, standard name is correct.

Change doc wording to match:
`Calculated as imported energy (Eimp) - exported energy (Eexp) - net increase in stored imported energy (Es)`

File: BuildingSync.xsd [hunk 4]
```diff
@@ -4076,6 +4076,78 @@
           </xs:simpleContent>
         </xs:complexType>
       </xs:element>
+      <xs:element name="BuildingEnergyUse" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>The annual amount of all the energy the building consumes onsite. Calculated as imported energy + onsite renewable energy - exported energy - net increase in stored energy (per ASHRAE 105-2014 Figure 5.6). (kBtu)</xs:documentation>
```
Comment (corymosiman12) [inline BuildingSync.xsd:4081]: Same as above:
`Calculated as imported energy (Eimp) + onsite renewable energy (Eg) - exported energy (Eexp) - net increase in stored imported energy (Es)`...

**Pull request merged**